src/sinks/aws_s3.rs

```diff
@@ -586,13 +590,15 @@ mod integration_tests {
     use std::io::{BufRead, BufReader};
     use vector_core::event::{BatchNotifier, BatchStatus, BatchStatusReceiver, LogEvent};
 
-    const BUCKET: &str = "router-tests";
-
     #[tokio::test]
     async fn s3_insert_message_into() {
         let cx = SinkContext::new_test();
 
-        let config = config(1000000).await;
+        let bucket = uuid::Uuid::new_v4().to_string();
```

Comment:
I refactored these tests a bit to always create a new bucket to avoid any pollution and separate the config creation from the bucket creation.